.github/workflows/installer-macos.yml

```diff
@@ -0,0 +1,70 @@
+on:
+  pull_request:
+  release:
+    types:
+      - created
+
+name: Create macOS App Bundle and DMG
+
+jobs:
+  build:
+    name: macOS App Bundle
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        build_type: ['Lite', 'Full']
+    defaults:
+      run:
+        shell: bash -l {0}
+        working-directory: ${{ github.workspace }}/installers/macOS
+    env:
+      LITE_FLAG: ${{ matrix.build_type == 'Lite' && '--lite' || '' }}
+      DMGNAME: ${{ matrix.build_type == 'Lite' && 'Spyder-Lite.dmg' || 'Spyder.dmg' }}
+      DISTDIR: ${{ github.workspace }}/dist
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+#      - name: Install Python
+#        uses: actions/setup-python@v2
+#        with:
+#          python-version: 3.7.9
```

Comment:
If this is no longer necessary, please remove it.

Reply:
Okay, no problem.
Is there a reason for using 3.7.9? Should I pursue using specified Python versions?

Reply:
There's one, but only apply to Windows. The latest version Python version supported by [Pynsist](https://github.com/takluyver/pynsist/), the package we use to create our Windows installer, is 3.7.